### PR TITLE
Start Running Passing Win32 AppLifeCycle Tests in Foundation Pipeline

### PR DESCRIPTION
**Pull request by kythant**
Start Running Passing Win32 AppLifeCycle Tests in Foundation Pipeline

### CODE REVIEW

File: test/AppLifecycle/FunctionalTests.cpp [hunk 1]
```diff
@@ -60,8 +60,8 @@ namespace ProjectReunionCppTest
             {
                 DeleteContentFile(c_testDataFileName_Packaged);
```
Comment (aeloros) [inline test/AppLifecycle/FunctionalTests.cpp:61]: nit: This can be commented out as well if you want.

**Review by EHO-makai — COMMENTED** [review]
Looks fine to me, Scott has the controlling approval.

File: test/AppLifecycle/FunctionalTests.cpp [hunk 2]
```diff
@@ -89,38 +89,38 @@ namespace ProjectReunionCppTest
...
-        }
+        //     VERIFY_IS_NOT_NULL(AppLifecycle::GetActivatedEventArgs());
+        // }
 
-        TEST_METHOD(GetActivatedEventArgsForLaunch)
-        {
-            BEGIN_TEST_METHOD_PROPERTIES()
-                // Run this test for both PackagedWin32 and Win32.
-                TEST_METHOD_PROPERTY(L"RunAs", L"{UAP,InteractiveUser}")
+        // TEST_METHOD(GetActivatedEventArgsForLaunch)
```
Comment (aeloros) [inline test/AppLifecycle/FunctionalTests.cpp:106]: I don't think we should be commenting out this valid test.

**Review by aeloros — CHANGES_REQUESTED** [review]

File: test/AppLifecycle/FunctionalTests.cpp (hunk 2 above)
Comment (kythant) [reply test/AppLifecycle/FunctionalTests.cpp:106]: "// Run this test for both PackagedWin32 and Win32."

I just noticed this comment. These were one of the tests that was failing on the pipeline so I am guess maybe it failed for the packaged version. I will try disabling just the packaged portion.

**Comment by aeloros** [comment]
Try changing the property instead.  You would change it to be the below.  This would change it to only run for Win32.  That’s a good point.  We may have issues in the pipeline that stop TAEF from using RunAs:UAP that need to be fixed (elevated privs?).


TEST_METHOD_PROPERTY(L"RunAs", L"InteractiveUser")

**Review by aeloros — APPROVED** [review]

**Pull request merged**